{"report":{"reportPrefix":"","assertionsCount":0,"lastError":{"code":"ECONNREFUSED","sessionCreate":true,"showTrace":false,"reportShown":true},"skippedAtRuntime":["Demo test orangeHRM"],"skippedByUser":[],"skipped":["Demo test orangeHRM"],"time":"0.001000","timeMs":1,"completed":{},"completedSections":{"__global_afterEach_hook":{"time":"0.001000","assertions":[],"commands":[],"passed":0,"errors":0,"failed":0,"skipped":0,"tests":0,"status":"fail","startTimestamp":1696576890786,"httpOutput":[["2023-10-06T07:21:30.542Z","  Request POST /session  ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: \u001b[32m&#39;firefox&#39;\u001b[39m } }\n  }"],["2023-10-06T07:21:30.656Z","  Request POST /session  (retry 1) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: \u001b[32m&#39;firefox&#39;\u001b[39m } }\n  }"],["2023-10-06T07:21:30.762Z","  Request POST /session  (retry 2) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: \u001b[32m&#39;firefox&#39;\u001b[39m } }\n  }"]],"timeMs":1,"endTimestamp":1696576890787}},"errmessages":[" \u001b[31mAn error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if \"start_process\" is set to \"true\".\u001b[39m"],"testsCount":0,"skippedCount":1,"failedCount":0,"errorsCount":1,"passedCount":0,"group":"","modulePath":"/home/knoldus/Documents/New Nightwatch_Project/nightwatch/LoginTestCase.ts","startTimestamp":"Fri, 06 Oct 2023 07:21:30 GMT","endTimestamp":"Fri, 06 Oct 2023 07:21:30 GMT","sessionCapabilities":{"browserName":"firefox","name":"OrangeHRM"},"sessionId":"","projectName":"","buildName":"","testEnv":"default","isMobile":false,"status":"fail","host":"localhost","name":"OrangeHRM","tags":["demo"],"tests":0,"failures":0,"errors":1,"httpOutput":[["2023-10-06T07:21:30.542Z","  Request POST /session  ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: <span style=\"color:#0A0\">&#39;firefox&#39;<span style=\"color:#FFF\"> } }\n  }</span></span>"],["2023-10-06T07:21:30.656Z","  Request POST /session  (retry 1) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: <span style=\"color:#0A0\">&#39;firefox&#39;<span style=\"color:#FFF\"> } }\n  }</span></span>"],["2023-10-06T07:21:30.762Z","  Request POST /session  (retry 2) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: <span style=\"color:#0A0\">&#39;firefox&#39;<span style=\"color:#FFF\"> } }\n  }</span></span>"]],"rawHttpOutput":[["2023-10-06T07:21:30.542Z","  Request POST /session  ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: 'firefox' } }\n  }"],["2023-10-06T07:21:30.656Z","  Request POST /session  (retry 1) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: 'firefox' } }\n  }"],["2023-10-06T07:21:30.762Z","  Request POST /session  (retry 2) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: 'firefox' } }\n  }"]],"globalErrorRegister":[" \u001b[31mAn error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if \"start_process\" is set to \"true\".\u001b[39m"]},"name":"LoginTestCase","httpOutput":[],"systemerr":" An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if \"start_process\" is set to \"true\"."}